pyproject.toml

```diff
@@ -41,6 +41,9 @@ test = [
     # Needed for testing asyncutil
     'async_solipsism==0.5.0',
 
+    # Needed for language server
+    'pygls~=1.3.1',
+
```

Comment:
I'm not sure where to specify this. It should not be packaged into edgedb-server and it is not really needed for tests.

Reply:
Put it into another feature, not `test`: 

```toml
lsp = [
    pygls~=1.3.1
]
```